displayer.go

```diff
@@ -1,9 +1,12 @@
 package plugin
 
+import "net/url"
+
 // Displayer is the interface plugins should implement to show a text to the user.
 // The text will appear on the plugin details page and can be multi-line.
 // Markdown syntax is allowed. Good for providing dynamically generated instructions to the user.
+// Location is the current location the user is accessing the API from, nil if not recoverable.
```

Comment:
So it will be https://somedomain.com/plugin/5/display? Maybe add an explicit example how it will look like.

Reply:
Okay I have added an example. Though I think most examples should reside in the website documentation with some contexts in plugins.

Reply:
Okay, I got what you meant:) I have commited you suggestions.